source/numbers.rst

```diff
@@ -42,6 +42,7 @@ You just type them as literals::
 Questions
 ---------
```

Comment:
Good suggestion! Another issue, is I have two question `1.`s on lines 45 and 46 😞